### DISCUSSION TITLE
Ansible Badges: A Better Way To Recognise Community Contributions

### DISCUSSION THREAD

**Discussion by Ompragash** [ansible-community/ansible-badges#3] [Show and tell]
Hi Everyone,

In our last Contributor Summit, we talked about "Ansible Badges" which is a new initiative and with that, we wanted to value and recognize the achievements of our contributors by providing a Digital Credential ([Open Badge Standard](https://support.badgr.com/en/knowledge/what-are-open-badges)).

To start with, we are first planning to reward Meetup contributions and come up with the below two digital badge designs for the same. 

As we do everything in a community way, we wanted our community members to finalize a design by replying to this discussion with 👍 /👎 emojis.

Feel free to comment your thoughts.

![Image 01-04-22 at 6 54 PM](https://user-images.githubusercontent.com/21008429/161272874-febb37d9-0ceb-4342-8158-f0b4414c6ad7.jpg)

**Comment by cybette** [comment]
I like both designs, although I have a slight preference for no. 1. Then again, maybe no. 2 is more badge-like. Help! I'm undecided... :stuck_out_tongue:

**Comment by gundalow** [comment]
I have a slight preference for the first one. If, for some badges, we get them physically printed as stickers do that help narrow the choice?

**Reply by bbaassssiiee** [reply]
I like number 2 for its color scheme, but it's more a tag than a badge. Number 1 is more a badge, but it's pale in comparison.

**Reply by amott-rh** [reply]
I prefer the 1st design but the darker colour of the 2nd badge - it makes the fairly pale Ansible logo stand out more.

**Comment by gundalow** [comment]
Thanks for all your feedback.
The first set of badges are in the process of being issued, they are for Meetup Organizer, and Meetup Speaker

https://badgr.com/public/issuers/LxI9xTZ5Qee4Luk5biGKWg/badges
